packages/xo-server-perf-alert/src/index.js

```diff
@@ -171,6 +171,15 @@ export const configurationSchema = {
             description: 'When enabled, all running hosts will be considered for the alert.',
             default: false,
           },
+          excludeUuids: {
+            description: 'List of hosts to exclude. Can be combined with "All running hosts"',
+            title: 'Exclude hosts',
+            type: 'array',
+            items: {
+              type: 'string',
+              $type: 'Host',
+            },
+          },
```

Comment:
That is not what was specified in the card.

Feel free to ping me if you want to discuss it.